osfoffline/settings/defaults.py

```diff
@@ -13,7 +13,7 @@
 FILE_BASE = 'https://staging-files.osf.io'
 
 # Interval (in seconds) to poll the OSF for server-side file changes
-POLL_DELAY = 5 * 60  # seconds
+POLL_DELAY = 1 * 60  # seconds (set to 1 min now for QA purpose, #todo change it back)
```

Comment:
This doesn't need to be committed (it can just be done when building the deployed version, changed locally)